### PR TITLE
Feat/migrate pyrdfj2

### PR DESCRIPTION
**Pull request by JotaFan**

### CODE REVIEW

File: pysubyt/j2/generator.py [hunk 1]
```diff
@@ -30,6 +30,9 @@ def __init__(self, templates_folder: str):
             ),
         )
         self._templates_env.globals = Functions.all()
+        # Since  the change ttl -> xsd, and removal of ttl to filters
+        # But this seems a bit redundact.
+        self._templates_env.globals.update(Filters.all())
```
Comment (bulricht) [inline pysubyt/j2/generator.py:35]: better handle this in pyrdfj2? @marc-portier

Comment (JotaFan) [reply pysubyt/j2/generator.py:35]: https://github.com/vliz-be-opsci/pyrdfj2/issues/9

Comment (marc-portier) [reply pysubyt/j2/generator.py:35]: joining with Bram on this one

**Comment by JotaFan** [comment]
it needs a new release from pyrdfj2

**Review by bulricht — APPROVED** [review]

**Review by cedricdcc — APPROVED** [review]

**Pull request merged**